crates/uv/tests/it/lock.rs

```diff
@@ -9669,6 +9669,121 @@ fn lock_find_links_explicit_index() -> Result<()> {
     Ok(())
 }
 
+#[test]
+fn lock_find_links_relative_url() -> Result<()> {
```

Comment:
Can you add a description what this test is checking? It's not clear to me from the name